frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -128,4 +194,63 @@ const ManagePatients = ({ activeFacilityId, canEditUser }: Props) => {
   );
 };
 
+const ManagePatients = (
+  props: Omit<Props, InjectedQueryWrapperProps | "pageCount">
+) => {
+  const activeFacilityId = useSelector(
+    (state) => (state as any).facility.id as string
+  );
+
+  const {
+    data: totalPatients,
+    loading,
+    error,
+    refetch: refetchCount,
+  } = useQuery(patientsCountQuery, {
+    variables: { facilityId: activeFacilityId, showDeleted: false },
+  });
+
+  if (activeFacilityId.length < 1) {
+    return <div>"No facility selected"</div>;
+  }
+
+  if (loading) {
+    return <p>Loading</p>;
+  }
+  if (error) {
+    throw error;
+  }
+
+  const entriesPerPage = props.entriesPerPage || 20;
+  const pageCount = Math.ceil(totalPatients.patientsCount / entriesPerPage);
+  const pageNumber = props.currentPage || 1;
+  return (
+    <QueryWrapper<Props>
+      query={patientQuery}
+      queryOptions={{
+        variables: {
+          facilityId: activeFacilityId,
+          pageNumber: pageNumber - 1,
+          pageSize: entriesPerPage,
+          showDeleted: props.showDeleted || false,
+        },
+      }}
+      onRefetch={refetchCount}
+      Component={DetachedManagePatients}
+      componentProps={{
+        ...props,
+        pageCount,
+      }}
+    >
+      <Pagination
+        baseRoute="/patients"
+        currentPage={pageNumber}
+        entriesPerPage={entriesPerPage}
+        totalEntries={totalPatients.patientsCount}
+        className="prime-home padding-bottom-5"
+      />
```

Comment:
The pagination also does not match the mock
![Screen Shot 2021-02-26 at 9 59 10 AM](https://user-images.githubusercontent.com/53869143/109316145-532e7080-7819-11eb-93d2-20be95587b84.png)

From the eng ready prototype the pagination should be part of the card containing the patient list
![Screen Shot 2021-02-26 at 9 58 53 AM](https://user-images.githubusercontent.com/53869143/109316085-414ccd80-7819-11eb-9f67-984b6790b365.png)

Reply:
Originally we had the blue circle. The problem is that if the blue circle is around the 22 it has to be wider, and in order to continue to be circular it has to be taller as well. So the circle size has to be proportional to the number's width meaning the circle size changes or else you end up with ovals. If you want it to look like the mock for the `1` case and ovals are fine then we can change it back.

Reply:
I'm fine with leaving out the blue circle for now, but can you make the active page bold? Right now it looks like it's only a color change. USWDS does not currently have pagination so we based our design based on this [draft design](https://github.com/uswds/uswds/issues/959). I believe pagination will be coming to USWDS soon so we may want to update to use their version soon anyways.

Reply:
@hmyers-cms The active page is bold already. I based the layout on [this code](https://codepen.io/USWDS/pen/eVzGzq) which is based on the USWDS draft design. Maya's screenshot in the ticket from 3 years ago shows two page numbers at the end, and the code in the Codepen only had one. Also there's no mobile implementation in that code. Not sure where their current design is.

Here's what it looks like when the number is big, if that's okay I can change it.
![Screen Shot 2021-02-26 at 12 15 36 PM](https://user-images.githubusercontent.com/40182224/109332710-7ca4c780-782c-11eb-9cb2-f16802c40c63.png)

Since there are no React components, even when the USWDS design is shipped we'll need to implement all the page display logic ourselves or wait for the `uswds-react` package to implement it. (Or work with them to build it?)